python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1326,3 +1326,32 @@ def test_is_month_start(data, dtype):
     got = gs.dt.is_month_start
 
     assert_eq(expect, got)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        [
+            "2020-05-31",
+            None,
+            "1999-12-01",
+            "2000-12-21",
+            None,
+            "1900-02-28",
+            "1800-03-14",
+            "2100-03-10",
+            "1970-01-01",
+            "1969-12-11",
```

Comment:
It might be a good idea to add `2020-09-29` here as a leap day test.

Reply:
Just added a leap day test.